vms/platformvm/block/executor/proposal_block_test.go

```diff
@@ -573,6 +579,10 @@ func TestBanffProposalBlockUpdateStakers(t *testing.T) {
 						Subnet: subnetID,
 					},
 					[]*secp256k1.PrivateKey{preFundedKeys[0], preFundedKeys[1]},
+					walletcommon.WithChangeOwner(&secp256k1fx.OutputOwners{
+						Threshold: 1,
+						Addrs:     []ids.ShortID{ids.ShortEmpty},
+					}),
```

Comment:
@dhrubabasu I believe we have to explicitly add the change addresses to avoid flaking here.
This UT is tricky because it tests a case where two stakers share the same end time.
The one evicted will be the one with the smallest TxID, which depends on every details of the transaction creating the staker.
If we don't declare the change address explicitly, any address from the funding keys will be picked. Since we have cases where multiple keys are specified, we have the randomness causing flaking.
The change address does not have to be empty, it can be anything fixed. I picked empty to try and signal the change address does not really matter (it's enough it does not change run by run).
I also added a comment to hopefully simplify maintenance.